core/ui/UIRichText.h

```diff
@@ -64,14 +63,14 @@ class AX_GUI_DLL RichElement : public Ref
      * @js ctor
      * @lua new
      */
-    RichElement(){};
+    RichElement() = default;
 
     /**
      * @brief Default destructor.
      * @js NA
      * @lua NA
      */
-    virtual ~RichElement(){};
+    ~RichElement() override = default;
```

Comment:
default destructor can be removed?

Reply:
Yes, a few them like that in the code. I'll clean them all up if you don't mind.